torch/csrc/jit/passes/quantization.cpp

```diff
@@ -413,30 +441,40 @@ class InsertObserversHelper {
   // These are the IR patterns we match to skip inserting observers.
   // They are compiled once on construction and used repeatedly within
   // the pass.
-  const PatternInfo conv_functional_relu = PatternInfo::parse_from_str(R"(
+  // 1. Module - Function
+  const PatternInfo mf_conv_functional_relu = PatternInfo::parse_from_str(R"(
 graph(%self, %input, %inplace):
     %relu = prim::Constant[name="relu"]()
-    %conv = match::module[name="Conv2d"](%self)
-    %intermediate_val = prim::CallMethod[name="forward"](%conv, %input)
-    %r = prim::CallFunction(%relu, %intermediate_val, %inplace)
-    return (%r) )");
-  const PatternInfo conv_relu_module = PatternInfo::parse_from_str(R"(
+    %first_module = match::module[name="Conv2d"](%self)
+    %first_output = prim::CallMethod[name="forward"](%first_module, %input)
+    %second_output = prim::CallFunction(%relu, %first_output, %inplace)
+    return (%second_output) )");
+  // 3. Module - Module
+  const PatternInfo mm_conv_relu = PatternInfo::parse_from_str(R"(
 graph(%self, %input):
-    %conv = match::module[name="Conv2d"](%self)
-    %intermediate_val = prim::CallMethod[name="forward"](%conv, %input)
-    %relu = match::module[name="ReLU"](%self)
-    %r = prim::CallMethod[name="forward"](%relu, %intermediate_val)
-    return (%r) )");
-  const PatternInfo matmul_add = PatternInfo::parse_from_str(R"(
+    %first_module = match::module[name="Conv2d"](%self)
+    %first_output = prim::CallMethod[name="forward"](%first_module, %input)
```

Comment:
Does this handle the case where you call the module, rather than calling forward directly?

Reply:
I think call the module is same as calling forward method: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L540